admin/create-amqp-extension

```diff
@@ -6,16 +6,20 @@ set -e -u
 source "$(dirname "${BASH_SOURCE[0]}")/lib/common.inc.bash"
 
 HELP=$(cat <<EOH
-Usage: $SCRIPT_NAME
+Usage: $SCRIPT_NAME [--help]
 
 Create 'amqp' extension (with broker) in the database.
 EOH
 )
 
-if [ $# -ne 0 ]
+if [[ $# -ne 0 && $1 =~ -*h(elp)? ]]
 then
-  echo >&2 "$SCRIPT_NAME: too many arguments"
-  echo >&2 "$HELP"
+  echo "$HELP"
+  exit
+elif [[ $# -ne 0 ]]
+then
```

Comment:
```suggestion
elif [[ $# -ne 0 ]]; then
```

Reply:
Not needed, semicolons are avoided as much as possible. Removed an errant one in https://github.com/metabrainz/musicbrainz-docker/pull/131/commits/9aa4495434916e65bf136218079e8c754de83dc0.

(Just coding style anyway.)